Jonathan-jonathankern/Homework-3/object-homework/app.js

```diff
@@ -0,0 +1,89 @@
+/////////////////////////////////////////////////////////////
+
+///////////////////////// Challenge /////////////////////////
+
+/////////////////////////////////////////////////////////////
+
+// Write your answers inside this file, at the places where it's indicated by the comments.
+
+
+// 1. Suppose that we wanted to create a browser-based calendar program for keeping track of things. In comments, list at least three reasonable abstractions that you might use to build this program; for each abstraction, list out at least two properties and behaviors that it would make sense for that abstraction to have. Below is an example - please write your answer in the same format.
+  // 'Car'
+  // Description: Our app manages car sales.
+  // Every 'car' has
+  // - a make
+  // - a model
+  // - a year
+  // Every 'car' can
+  // - drive
+  // - brake
+  // - park
+
+// Answer Starts Here
+
+/*'Calendar'
+Description: Browser-based calendar app
+Every 'calendar' has
+  - a year
+  - a month
+  - a week
+  - a day
+Every 'calendar' can
+  - create events
+  - edit events
+  - save events
+*/
+
+// Answer Ends Here
+
+
+// 2. Create an Object literal that lines up with the following description. Store it in the variable 'pet_owner', below. Be sure to give it reasonable values for each of its properties.
+
+  // 'Owner'
+  // Description: We are making an app for a veterinary clinic - it manages pets' vaccinations.
+  // Every 'owner' has:
+  // - a name
+  // - an address
+
+let pet_owner;
+
+// Answer Starts Here
+pet_owner = {
+	name: "John Smith",
+	address: "123 Main St., San Francisco, CA, 94102"
```

Comment:
This totally works. I also encourage you to also think about different ways the data could be represented and stored -- for instance, what if the value of the address property was itself an object, which contained properties for street address, city, state, and zip?